public/css/stylesheet.css

```diff
@@ -0,0 +1,275 @@
+/* - - - - -  Landing Page CSS - - - - - */
```

Comment:
There's a lot going on in this file - consider splitting it up into multiple files for ease of maintenance.  You could create stylesheets for each of the sections you highlight, and either `link` them in the appropriate order, or (better) use tooling to combine and minify them.